test/uk/gov/hmrc/transitmovements/controllers/ArrivalsControllerSpec.scala

```diff
@@ -328,7 +328,7 @@ class ArrivalsControllerSpec
     "must return OK if arrivals were found" in {
       val response = MovementWithoutMessages.fromMovement(movement)
 
-      when(mockRepository.getMovements(EORINumber(any()), eqTo(MovementType.Arrival)))
+      when(mockRepository.getMovements(EORINumber(any()), eqTo(MovementType.Arrival), eqTo(None)))
```

Comment:
Add test cases to cover optional updatedSince parameter. 
Like below code available in same controller 
  "getArrivalMessages" - {

    val request = FakeRequest("GET", routes.ArrivalsController.getArrivalMessages(eoriNumber, movementId).url)

    "must return OK and a list of message ids if an arrival is found and messages match the receivedSince filter" in forAll(
      arbitrary[MessageResponse],
      Gen.option(arbitrary[OffsetDateTime])
    ) {
      (messageResponses, receivedSince) =>